CRM/Utils/Check/Component/Env.php

```diff
@@ -688,6 +688,54 @@ public function checkExtensions() {
     return $messages;
   }
 
+  /**
+   * @return CRM_Utils_Check_Message[]
+   */
+  public function checkScheduledJobLogErrors() {
+    $jobs = civicrm_api3('Job', 'get', [
+      'sequential' => 1,
+      'return' => ["id", "name", "last_run"],
+      'is_active' => 1,
+      'options' => ['limit' => 0],
+    ]);
+    $html = '';
+    foreach ($jobs['values'] as $job) {
+      $lastExecutionMessage = civicrm_api3('JobLog', 'get', [
+        'sequential' => 1,
+        'return' => ["description"],
+        'job_id' => $job['id'],
+        'options' => ['sort' => "id desc", 'limit' => 1],
+      ])['values'][0]['description'] ?? NULL;
+      if (!empty($lastExecutionMessage) && strpos($lastExecutionMessage, 'Failure') !== FALSE) {
+        $viewLogURL = CRM_Utils_System::url('civicrm/admin/joblog', "jid={$job['id']}&reset=1");
+        $html .= '<tr>
+          <td>' . ts('%1', [1 => $job['name']]) . ' </td>
+          <td>' . ts('%1', [1 => $lastExecutionMessage]) . '</td>
+          <td>' . ts('%1', [1 => $job['last_run']]) . '</td>
+          <td>' . ts('<a href=\'%1\'>View Job Log</a>', [1 => $viewLogURL]) . '</td>
```

Comment:
Can you do instead:

```
<td><a href="' . $viewLogURL . '"> . ts('View Job Log') . '</a></td>
```

This way it avoids adding a new string to translate (and HTML should be avoided when possible).

```
$ grep -r 'View Job Log'
CRM/Admin/Page/Job.php:          'name' => ts('View Job Log'),
```

Reply:
Updated with the above suggestion 👍